lua/drex/drawer.lua

```diff
@@ -59,8 +59,9 @@ end
 function M.open()
     local win = M.get_drawer_window()
     if not win then
+        local side = (config.options.drawer.side == 'left' and 'H' or 'L')
```

Comment:
My personal feeling would be that `left` is the "default" for most people which is why I would also rather check for `right` explicitly, so in case the user enters some other string into the config the default would be to open the drawer on the left side (which was also the default before)

Basically just "turning around" your check here :slightly_smiling_face: